azure-webapp-maven-plugin/README.md

```diff
@@ -1,182 +1,175 @@
     ```
     $ mvn azure-webapp:deploy
     ```
 
-<a name="samples"></a>
-## Samples	
-A few typical usages of Maven Plugin for Azure App Service Web Apps are listed at [Web App Samples](../docs/web-app-samples.md). You can choose one to quickly get started.
-
 <a name="goals"></a>
 ## Goals
 
-The Maven Plugin for Azure App Service Web Apps has only one goal: `azure-webapp:deploy`. 
-
-Goal | Description
---- | ---
-`azure-webapp:deploy` | Deploy artifacts or docker container image to an Azure Web App based on your configuration.<br>If the specified Web App does not exist, it will be created.
-## Authentication with Azure
-
-You can use the Azure CLI 2.0 for authentication. More authentication methods can be found [here](../docs/common-configuration.md).  
-
-1. Install the Azure CLI 2.0 by following the instructions in the [Install Azure CLI 2.0](https://docs.microsoft.com/cli/azure/install-azure-cli) article.
-
-2. Run the following commands to log into your Azure subscription:
-
-    ```
-    $ az login
-    $ az account set --subscription <subscription Id>
-    ```
-## Configurations
-
-Common configurations of all Maven Plugins for Azure can be found [here](../docs/common-configuration.md).
-
-The maven plugin supports two kinds of configurations V2 and V1 (deprecated). Specify the configuration 
-`<schemaVersion>V2</schemaVersion>` to use the V2 configuration.
-The configurations of the region, the Web App runtime and the deployment are different in V2 and V1.
-The common basic settings of the configuration are listed in the following table.
-
-Property | Required | Description | Version
----|---|---|---
-`<resourceGroup>` | true | Azure Resource Group for your Web App. | 0.1.0+
-`<appName>` | true | The name of your Web App. | 0.1.0+
-`<pricingTier>`* | false | The pricing tier for your Web App. The default value is **P1V2**.| 0.1.0+
-`<deploymentSlot>` | false | The deployment slot to deploy your application. | 1.3.0+
-`<appServicePlanResourceGroup>` | false | The resource group of the existing App Service Plan. If not specified, the value defined in `<resourceGroup>` will be used by default. | 1.0.0+
-`<appServicePlanName>` | false | The name of the existing App Service Plan. | 1.0.0+
-`<appSettings>` | false | Specifies the application settings for your Web App. | 0.1.0+
-`<stopAppDuringDeployment>` | false | To stop the target Web App or not during deployment. This will prevent deployment failure caused by IIS locking files. | 0.1.4+
-
-### V2 Configuration
-
-1. Region
-
-    The configuration `<region>` is required in V2 configuration, supported values could be found in the following.
-
-2. Runtime settings
+#### `azure-webapp:config`
+-  A command line tool providing a great experience for managing webapp deploy configuration, it will rewrite your pom config based on your input. Detail about the configuation could be found in the following.
+-  The command line will divide the config into 3 parts. 
```

Comment:
config -> configuration (Avoid using the abbreviation in the documents)

Reply:
OK